package.json

```diff
@@ -17,16 +17,16 @@
     "lint:lit": "lit-analyzer \"{components,directives,helpers,mixins,templates,test,tools}/**/*.js\" --strict",
     "lint:style": "stylelint \"**/*.js\"",
     "prepublishOnly": "node ./cli/verify-translations.js && npm run analyze",
-    "start": "es-dev-server --node-resolve --dedupe --watch --open",
-    "test": "npm run lint && npm run test:headless && npm run test:axe && npm run test:diff",
+    "start": "web-dev-server --open index.html --node-resolve",
+    "test": "npm run lint && npm run test:headless && npm run test:diff",
```

Comment:
Not running `aXe` tests anymore?

Reply:
Yeah - definitely will re-add once I figure out the configs more